http://bugs.gentoo.org/410565

Unknown type name errors with references to GObject, GObjectClass, GType and GTypeInfo

The errors started manifesting after upgrading to Xfce 4.10 pre1 libraries

--- xfburn/xfburn-settings.h
+++ xfburn/xfburn-settings.h
@@ -24,6 +24,7 @@
 #endif /* !HAVE_CONFIG_H */
 
 #include <glib.h>
+#include <glib-object.h>
 
 G_BEGIN_DECLS
 
